### PR TITLE
K-Atlas website updated files

### PR DESCRIPTION
**Pull request by kevinli11**

### CODE REVIEW

**Comment by fzxu** [comment]
do you have the screen shot for the update?

**Comment by kevinli11** [comment]
> do you have the screen shot for the update?

yes
K-Atlas homepage V1

**Comment by davemasselink** [comment]
looks ok to me, but I'll also wait until I can see a screenshot before approving

File: website/_includes/footer.html [hunk 1]
```diff
@@ -1,8 +1,15 @@
-<footer class="site-footer">
-  <div class="container" align="center">
-    <p style="font-size: 14px; font-weight: 600"><a href="{{ site.baseurl }}/about">About</a>&nbsp;&nbsp;&nbsp;&nbsp; <a href="{{ site.baseurl }}/contact">Contact</a>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://github.com/intuit/katlas">GitHub</a>&nbsp;&nbsp;&nbsp;&nbsp; </p>
-    <p style="margin-top: 15px; font-size: 14px;">&#169; 2018. <a href="https://katlas.io">{{ site.author.name }}</a>. <a rel="license" href="https://www.apache.org/licenses/LICENSE-2.0">Apache License</a>.</p>
-  </div>
+<footer class="site-footer" >
+
```
Comment (davemasselink) [inline website/_includes/footer.html]: @kevinli11, do you intend for all the extra whitespace in these changes?

File: website/assets/css/homepage.css [hunk 2]
```diff
@@ -49,6 +82,85 @@ h1, h2, h3, h4, h5, h6 {
...
+.feature-box .icon {
+  font-size:30px;
+  position:absolute;
+  left:15px;
+  top:15px;
+  width:30px;
+  text-align:center;
+  color:#3b99e0;
+}
+
```
Comment (davemasselink) [inline website/assets/css/homepage.css]: may want to clean-up extra whitespace here

File: website/assets/css/homepage.css [hunk 3]
```diff
@@ -60,31 +172,47 @@ h1, h2, h3, h4, h5, h6 {
...
+.feature-box .icon {
+  font-size:30px;
+  position:absolute;
+  left:15px;
+  top:15px;
+  width:88px;
+  text-align:center;
+  color:#3b99e0;
+}
+
```
Comment (davemasselink) [inline website/assets/css/homepage.css]: may want to clean-up extra whitespace here

File: website/assets/css/page.css [hunk 4]
```diff
@@ -10,6 +10,34 @@ h1, h2, h3, h4, h5, h6 {
...
+  background: #135CC0;
+  color: #FFFFFF;
+  max-width: 100%;
+  height:220px;
+  padding-bottom: 0px
+  margin-bottom: 1rem;
+  border-radius: 0!important;
+}
+
+
```
Comment (davemasselink) [inline website/assets/css/page.css]: may want to clean-up extra whitespace here

File: website/_includes/footer.html (hunk 1 above)
Comment (kevinli11) [reply website/_includes/footer.html]: @davemasselink : I don't think I need them.

File: website/assets/css/homepage.css (hunk 2 above)
Comment (kevinli11) [reply website/assets/css/homepage.css]: No, It doesn't need.

File: website/assets/css/homepage.css (hunk 3 above)
Comment (kevinli11) [reply website/assets/css/homepage.css]: No, It doesn't need.

File: website/assets/css/page.css (hunk 4 above)
Comment (kevinli11) [reply website/assets/css/page.css]: No, It doesn't need.

**Pull request merged**